### DISCUSSION TITLE
v3 livewire.js Not Found on production server

### DISCUSSION THREAD

**Discussion by faizananwerali** [filamentphp/filament#8151] [Help] [answered]
`GET https://domain.com/livewire/livewire.js?id=11c49d7e net::ERR_ABORTED 404 (Not Found)`

In Livewire v2, I could use this `php artisan livewire:publish --assets` to publish assets, not anymore.

and in composer.json

```json
        "post-update-cmd": [
            "@php artisan vendor:publish --tag=laravel-assets --ansi --force",
            "@php artisan vendor:publish --tag=livewire:assets --ansi --force",
            "@php artisan filament:upgrade"
        ],
```

but this is not working anymore, all the work I've done past two weeks is inaccessible because livewire.js is not found.

Can anybody please help me here

**Comment by danharrin** [comment]
No need to ask here, the Livewire repo is the correct place.

**Comment by faizananwerali** [comment, marked as answer]
Found my answer here [https://github.com/livewire/livewire/discussions/6328](https://github.com/livewire/livewire/discussions/6328)
